### PR TITLE
Swagger update

### PR DESCRIPTION
**Pull request by ralph**

### CODE REVIEW

File: lib/global_event_technology_api_client/configuration.rb [hunk 1]
```diff
@@ -125,7 +125,7 @@ class Configuration
     attr_accessor :force_ending_format
 
     def initialize
-      @scheme = 'https'
+      @scheme = 'http'
```
Comment (svenf1105) [inline lib/global_event_technology_api_client/configuration.rb]: pretty sure we want https

**Review by svenf1105 — CHANGES_REQUESTED** [review]
did you increase the version number of the gem? otherwise myomr won't update, we also have this version number in the gemfile

File: lib/global_event_technology_api_client/configuration.rb (hunk 1 above)
Comment (ralph) [reply lib/global_event_technology_api_client/configuration.rb]: Right, good catch! Fixed in [993523c](https://github.com/ramp106/global_event_technology_api_client/pull/3/commits/993523ce71389ca895e42ee19e5d3d0ec7abb462)

**Comment by ralph** [comment]
EDIT: Found the version specification, it's fixed in 66594af4af5f9be1fd5a5ae6322622d832a3a619.

**Review by svenf1105 — APPROVED** [review]

**Pull request merged**